### PR TITLE
add node record pod info

### PR DESCRIPTION
**Pull request by lengrongfu**
**What type of PR is this?**

**What this PR does / why we need it**:
Add `hami.io/bind-pod-info` annotation, to record current `allocate` device pod.

**Which issue(s) this PR fixes**:
Fixes # https://github.com/Project-HAMi/HAMi/issues/404

**Special notes for your reviewer**:

**Does this PR introduce a user-facing change?**:

### CODE REVIEW

File: pkg/util/util.go [hunk 1]
```diff
@@ -96,6 +103,19 @@ func GetPendingPod(node string) (*corev1.Pod, error) {
 	return nil, fmt.Errorf("no binding pod found on node %s", node)
 }
 
+func GetAllocatePodByNode(ctx context.Context, nodeName string) (*corev1.Pod, error) {
+	node, err := client.GetClient().CoreV1().Nodes().Get(ctx, nodeName, metav1.GetOptions{})
+	if err != nil {
+		return nil, err
+	}
+	if podInfo, ok := node.Annotations[nodelock.NodeLockBindPodInfo]; ok {
+		klog.Infof("node annotation key is %s, value is %s ", nodelock.NodeLockBindPodInfo, podInfo)
```
Comment (chaunceyjiang) [inline pkg/util/util.go]: Set the log level.

File: pkg/util/nodelock/nodelock.go [hunk 2]
```diff
@@ -28,11 +28,12 @@ import (
 )
 
 const (
-	NodeLockTime = "hami.io/mutex.lock"
-	MaxLockRetry = 5
+	NodeLockTime        = "hami.io/mutex.lock"
```
Comment (chaunceyjiang) [inline pkg/util/nodelock/nodelock.go]: I suggest reusing `NodeLockTime` so we don't have to maintain an additional field. Reusing `NodeLockTime` will also make the logic clearer.

**Comment by lengrongfu** [comment]
Use new annotation key, pod can success scheduler.
image


image

**Comment by lengrongfu** [comment]
@chaunceyjiang PTAL

**Pull request merged**